### PR TITLE
Clarify image-related services docs

### PR DESCRIPTION
**Pull request by Hixie**
...and make one of the comments accurate by fixing the code. Turns
out I forgot to add the 'silent' argument when I changed that comment.

### CODE REVIEW

File: packages/flutter/lib/src/services/image_stream.dart [hunk 1]
```diff
@@ -88,6 +88,10 @@ class ImageStream {
   ///
   /// This is usually done automatically by the [ImageProvider] that created the
   /// [ImageStream].
+  ///
+  /// This method can only be called once per stream. To have an [ImageStream]
+  /// represent multiple images over time, assign it a completer that
```
Comment (tvolkert) [inline packages/flutter/lib/src/services/image_stream.dart]: that represents

**Review by tvolkert — APPROVED** [review]
LGTM

File: packages/flutter/lib/src/services/image_stream.dart (hunk 1 above)
Comment (Hixie) [inline packages/flutter/lib/src/services/image_stream.dart]: why?

Comment (tvolkert) [inline packages/flutter/lib/src/services/image_stream.dart]: Oh, I misread it last night. Nevermind :-)

**Comment by Hixie** [comment]
Thanks. Will land on green.

**Pull request merged**